gcc/rust/expand/rust-macro-builtins-asm.cc

```diff
@@ -0,0 +1,21 @@
+//
+// Created by jjsm on 2/25/24.
```

Comment:
Please use the regular header with the license & boilerplate text instead. I guess it's coming from your editor?